pkgs/by-name/da/datatrove/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+}:
+python3Packages.buildPythonPackage rec {
+  pname = "datatrove";
+  version = "0.2.0";
```

https://nix.dev/guides/best-practices.html#recursive-attribute-set-rec
```suggestion
let
  version = "0.2.0";
in
python3Packages.buildPythonPackage {
  pname = "datatrove";
  inherit version;
```